pkg/ipamd/datastore/data_store_test.go

```diff
@@ -925,7 +925,6 @@ func TestGetIPStatsV4(t *testing.T) {
 		*ds.GetIPStats("4"),
 	)
 
-	// wait 30s (cooldown period)
 	time.Sleep(30 * time.Second)
```

Comment:
We should take this opportunity to fix this test, as sleeping for 30 seconds is not ideal. Maybe we can set the cooldown period lower to start? Or change the time